rs/bitcoin/kyt/src/providers.rs

```diff
@@ -0,0 +1,72 @@
+use bitcoin::Network;
+use ic_btc_interface::Txid;
+use ic_cdk::api::management_canister::http_request::{
+    CanisterHttpRequestArgument, HttpHeader, HttpMethod, TransformContext, TransformFunc,
+};
+
+pub fn create_request(
+    network: Network,
+    txid: Txid,
+    max_response_bytes: u32,
+) -> CanisterHttpRequestArgument {
+    match network {
+        Network::Bitcoin => btcscan_request(txid, max_response_bytes),
+        Network::Testnet => mempool_space_testnet_request(txid, max_response_bytes),
```

Comment:
These should later on be renamed to `mainnet_request` and `testnet_request`, which contain the logic to send requests to different providers.

Reply:
Agree. I will leave it to ticket XC-159